ModelCatalogueGenomicsPlugin/src/groovy/org/modelcatalogue/gel/export/RareDiseaseChangeLogXlsExporter.groovy

```diff
@@ -425,7 +436,7 @@ abstract class RareDiseaseChangeLogXlsExporter extends AbstractChangeLogGenerato
                 changes << (after ? "$key: $after" : '')
             }
 
-            if (subtype) {          // cache detail changes
+            if(this instanceof RareDiseasePhenotypeChangeLogXlsExporter) {            // cache detail changes
```

Comment:
this smells a bit, should use generic implementation method (the else branch) and in `RareDiseasePhenotypeChangeLogXlsExporter ` the special one but let it be for now